packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -1182,6 +1183,22 @@ describe('Checkout', () => {
                     // Assert
                     expect(loadCheckoutSpy).not.toHaveBeenCalled();
                 });
+
+                it('should call `getAddressFromLocalStorage` so we can pre-populate the guest checkout address', async () => {
+                    // Arrange & Act
+                    const getAddressFromLocalStorageSpy = jest.spyOn(VueCheckout.methods, 'getAddressFromLocalStorage');
+
+                    shallowMount(VueCheckout, {
+                        store: createStore(),
+                        i18n,
+                        localVue,
+                        propsData
+                    });
+                    await flushPromises();
+
+                    // Assert
+                    expect(getAddressFromLocalStorageSpy).toHaveBeenCalled();
+                });
```

Comment:
Not one for now, but for the future, because it's not something you're introducing now. We're unit testing `initialise`, yet we're not calling that method directly, so it's not really a unit test.